internal/datatransfer/monitor.go

```diff
@@ -257,3 +231,44 @@ func (m *Monitor) Update(configuration models.DeviceConfigurationMessage) error
 	m.SetStorage(s3sync)
 	return nil
 }
+
+func containsDataPaths(dc *models.DataConfiguration) bool {
+	return dc != nil &&
+		(dc.Egress != nil && len(dc.Egress) > 0 ||
+			dc.Ingress != nil && len(dc.Ingress) > 0)
+}
+
+func syncDataConfiguration(syncWrapper FileSync, dataConfig *models.DataConfiguration, hostPath string) error {
```

Comment:
It syncs data, not the the configuration; please rename.

Reply:
renamed to `syncData`